src/utils.ts

```diff
@@ -88,3 +88,15 @@ export async function getAvailableOptions(
 
   return availableOptions;
 }
+
+export function deepMerge(target: any, source: any): any {
```

Comment:
I found native deepMerge in deno, try it

https://jsr.io/@std/collections/doc/~/DeepMerge